statsmodels/nonparametric/smoothers_lowess.py

```diff
@@ -10,7 +10,7 @@
 import numpy as np
 from ._smoothers_lowess import lowess as _lowess
 
-def lowess(endog, exog, frac=2.0/3.0, it=3, delta=0.0, is_sorted=False,
+def lowess(endog, exog, frac=2.0/3.0, it=3, delta=0.0, xvals=None, is_sorted=False,
```

Comment:
The new optional should, unfortunately, go at the end.  I cold also live with making everything after (and including) `it=3` keyword only, so that `frac=2.0/3.0, *, it=3,...`.

Reply:
This breaks code but in a way that is loud and obvious.  Leaving it were it is could like to hard to trace breaks.